core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -77,7 +77,12 @@ object StaticFile {
         .flatMap(fromURL(_, req)))
   }
 
-  def fromURL[F[_]](url: URL, req: Option[Request[F]] = None)(implicit
+  def fromURL[F[_]: Sync: Files](
```

Comment:
this `Files` constraint is not used, so it stopped compiling in CI
then since that is not used, perhaps we need to remove that in fromResource as well.
it should probably also be rebased on `main` or `series/0.22`

Reply:
https://github.com/http4s/http4s/pull/4879/checks?check_run_id=2680925168#step:8:96
and
https://github.com/http4s/http4s/pull/4879/checks?check_run_id=2680925109#step:8:24

Reply:
Gotcha, thanks hamnis! I'll try and fix it in a couple hours or so 😄